Resources/Prototypes/_White/Catalog/uplink_catalog.yml

```
@@ -46,6 +46,23 @@
   categories:
   - UplinkWeapons
   conditions:
+  - !type:StoreWhitelistCondition
```

yamllint

[warning] wrong indentation: expected 4 but found 2

(indentation)